modules/somoaudienceBidAdapter.js

```diff
@@ -1,5 +1,6 @@
 import {getTopWindowReferrer, getTopWindowLocation} from 'src/utils';
 import { registerBidder } from 'src/adapters/bidderFactory';
+const realvuAnalyticsAdapter = require('modules/realvuAnalyticsAdapter.js');
```

Comment:
You can't pull in other modules from an adapter. There's no build-time way to do this, and run-time loading of code isn't allowed.

It appears you're trying to link Somo and RealVu... there's not a precedence for this. Please explain the background?